keps/sig-apps/2360-optional-service-environment-variables/kep.yaml

```diff
@@ -0,0 +1,19 @@
+title: Optional Service Environment Variables
+kep-number: 2360
+authors:
+  - "@bradhoekstra"
+  - "@kongslund"
+owning-sig: sig-apps
+participating-sigs:
+reviewers:
+  - TBD
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2018-09-25
+last-updated: 2018-09-25
+status: provisional
```

Comment:
It seems it was implemented, but we can probably update that later.

Reply:
OKay